### PR TITLE
[kws] add code for plotting det curve

### PR DESCRIPTION
**Pull request by mlxu995**
image

image

### CODE REVIEW

File: kws/bin/plot_det_curve.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    plt.rcParams['ytick.direction'] = 'in'
+    plt.rcParams['text.usetex'] = True
+    plt.rcParams['font.size'] = 12
+
+    for index, keyword in enumerate(keywords):
+        stats_file = os.path.join(stats_dir, 'stats.' + str(index) + '.txt')
+        values = load_stats_file(stats_file)
+        plt.plot(values[:, 0], values[:, 1], label=keyword)
+
+    plt.xlim([0, 5])
```
Comment (robin1001) [inline kws/bin/plot_det_curve.py:45]: please set xlim and ylim by parameter

File: kws/bin/plot_det_curve.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    plt.figure(dpi=200)
+    plt.rcParams['xtick.direction'] = 'in'
+    plt.rcParams['ytick.direction'] = 'in'
+    plt.rcParams['text.usetex'] = True
+    plt.rcParams['font.size'] = 12
+
+    for index, keyword in enumerate(keywords):
+        stats_file = os.path.join(stats_dir, 'stats.' + str(index) + '.txt')
+        values = load_stats_file(stats_file)
+        plt.plot(values[:, 0], values[:, 1], label=keyword)
```
Comment (robin1001) [inline kws/bin/plot_det_curve.py:43]: how about the color? is it the same color for multiple wakeup words?

File: kws/bin/plot_det_curve.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        required=True,
+        help=('keywords, must in the same order as in "dict/words.txt", ' +
+              'separated by ", "')
+    )
+    parser.add_argument('--stats_dir', required=True, help='dir of stats files')
+    parser.add_argument(
+        '--figure_file',
+        required=True,
+        help='path to save det curve')
+    parser.add_argument('--xlim', required=True, help='range of X-axis')
```
Comment (robin1001) [inline kws/bin/plot_det_curve.py:75]: It's better to  make the four parameters optional.

File: kws/bin/plot_det_curve.py (hunk 2 above)
Comment (mlxu995) [reply kws/bin/plot_det_curve.py:49]: for multiple wake-up words (more than one curve), matplotlib will set each a different color by default.
image

File: kws/bin/plot_det_curve.py (hunk 3 above)
Comment (mlxu995) [reply kws/bin/plot_det_curve.py:75]: considering that normal det curves tend to be in the lower left corner, I make the start of the X-axis and Y-axis constant.

Comment (robin1001) [reply kws/bin/plot_det_curve.py:75]: It's better to give a default value as reference for the users.

**Comment by robin1001** [comment]
Great job!

**Review by robin1001 — APPROVED** [review]

**Pull request merged**